modules/ding_provider/connie_search/includes/connie_search.search.inc

```diff
@@ -274,3 +251,16 @@ function connie_search_search_autocomplete($query) {
 
   return $items;
 }
+
+/**
+ * Quotes a search string to make it compatible with the provider.
+ *
+ * @param string $search_string
+ *   The search string.
+ *
+ * @return string
+ *   The quoted search string.
+ */
+function connie_search_search_search_quote($search_string) {
```

Comment:
Not that it is ultimately a great excuse but the call is called "quote" because it's now the guts of _ting_search_quote - I'm not feeling like renaming it and all uses of it at this point :/